rlclientlib/logger/async_batcher.h

```diff
@@ -169,7 +169,7 @@ namespace reinforcement_learning { namespace logger {
     _queue_mode(queue_mode)
   {}
 
-  template<typename TEvent, template<typename> typename TSerializer>
+  template<typename TEvent, template<typename> class TSerializer>
```

Comment:
Why change this to class?

Reply:
Compiler in linux had issues with template template arguments.